index.js

```diff
@@ -246,6 +250,11 @@ class CreateCertificatePlugin {
    * Prints out a summary of all domain manager related info
    */
   certificateSummary() {
+    const region = this.options.region ? this.options.region : this.serverless.service.provider.region;
+    if (region.startsWith('cn-')) {
```

Comment:
2nd use of cn- -> what about a constant or list of "unsupported regions"

Reply:
Yes that is also an option.

Reply:
I remembered why I decided to not go with "unsupported regions": Yes we can have a list of regions but what if AWS decides to add a new region? We'd have to maintain this list. We can assume that AWS will ship every new region with Certificate Manager as they have in the past. This is just a China specific issue as the Chinese government does not allow straight forward ways of issuing a certificate. This code can be expected to be future-proof as you don't have to consider new Chinese regions (they will be excluded automatically) and new other regions (they will be supported automatically).

Reply:
well, it is hard to predict the future :) But if aws decides to add a new region and it supports acm as in the past the code still works. If Aws decides to add a new region that does not support acm we have to add it anyways. I am not saying you should abandon the region wildcard approach => cn- 
I would just much rather have a list that has a name e.g. unsupportedRegions than a magical string we use two times